### PR TITLE
Rebuilt storybook native + fix Android time picker

### PR DESCRIPTION
**Pull request by xuan9230**
1. Rebuilt storybook native with latest react native version (0.64), fixed several issues.
2. TimePicker: converted from flow to ts and use the new native component for android.

### CODE REVIEW

File: packages/bappo-components/src/components/TimePicker/TimePicker.android/index.tsx [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    }
+
+    return <ValueText>{_timeStr}</ValueText>;
+  }
+
+  function blur() {
+    pickerRef.current && pickerRef.current.blur();
+  }
+
+  function focus() {
```
Comment (Jonathanliu525) [inline packages/bappo-components/src/components/TimePicker/TimePicker.android/index.tsx:60]: Hi Stan, it seems like this`focus`  is not used in the code

File: packages/bappo-components/src/components/TimePicker/TimePicker.android/index.tsx [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    onBlur: () => {
+      setShowPicker(false);
+      onBlur && onBlur();
+    },
+    onFocus: () => {
+      setShowPicker(true);
+      onFocus && onFocus();
+    },
+    readOnly,
+    ref: pickerRef,
```
Comment (Jonathanliu525) [inline packages/bappo-components/src/components/TimePicker/TimePicker.android/index.tsx:78]: Hi Stan, I am sorry I also have a question here about the ref. In the component <PickerNative />, it seems there is no place to use this ref property

**Review by Jonathanliu525 — COMMENTED** [review]
Hi,Stan. I have tested this new timepicker on Android simulator and it works fine on my machine. But I have two questions listed in blow

File: packages/bappo-components/src/components/TimePicker/TimePicker.android/index.tsx (hunk 1 above)
Comment (xuan9230) [reply packages/bappo-components/src/components/TimePicker/TimePicker.android/index.tsx:60]: Yes it's only open for future use.

File: packages/bappo-components/src/components/TimePicker/TimePicker.android/index.tsx (hunk 2 above)
Comment (xuan9230) [reply packages/bappo-components/src/components/TimePicker/TimePicker.android/index.tsx:78]: PMed you

**Pull request merged**